sql/KOD-30.1.txt

```diff
@@ -0,0 +1,18 @@
+DROP PROCEDURE IF EXISTS UserName;
```

Comment:
Co to jest?

Reply:
A nie lepiej takie coś zostawiać, gdyby przyszedł ktoś i cos zmieniał w tej procedurze?

Reply:
To powinno byc w tym samym miejscu gdzie definicja procedury UserName

Reply:
To nazwij ta procedure GetUserById